shared/profile/post-proof.desktop.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+import React from 'react'
+import {Box, Text, Button, Icon, PlatformIcon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './post-proof'
+
+const Render = (props: Props) => {
+  const {platform, platformUserName, platformSubtitle, descriptionView, descriptionText, proofText, proofAction, proofActionIcon, proofActionText, noteText, onCancel, onCancelText, onComplete, onCompleteText, isOnCompleteWaiting, errorMessage} = props
```

Comment:
Note for others: we have to do this so flow understands these are `const`, if you destructure within the function arg spot flow (correctly) parses them as `var`s.

The problem with that is when you do `() => onFoo` flow isn't sure if `onFoo` was mutated to be not a function.